src/cmd/manage_nsfs.js

```diff
@@ -215,40 +215,31 @@ async function add_bucket(data) {
 }
 
 /** verify_bucket_owner will check if the bucket_owner has an account
+ * bucket_owner is the account name in the account schema
  * after it finds one, it returns the account id, otherwise it would throw an error
  * (in case the action is add bucket it also checks that the owner has allow_bucket_creation)
- * @param {string} bucket_owner account name
+ * @param {string} bucket_owner
  * @param {string} action
  */
 async function verify_bucket_owner(bucket_owner, action) {
-    let is_bucket_owner_exist = false;
-    let is_allow_bucket_creation = false;
-    let account_id;
+    // check if bucket owner exists
+    const account_config_path = get_config_file_path(accounts_dir_path, bucket_owner);
     const show_secrets = false;
```

Comment:
@shirady do we need a const here, we can directly pass `false`?

Reply:
@naveenpaul1 we don't need it, I think it is more readable than just `false`.
I will try to add it in the constant file, and see if we like the result.